### ISSUE TITLE
Private User Workspace

### ISSUE THREAD

**Issue by WaltRCodes** [ponder-lab/che#8]
Each student should have their own private workspace. This is already in Eclipse Che, but we will have to look at a way for multiple users to access it and keep their work private.

**Comment by khatchad** [comment]
> Each student should have their own private workspace. This is already in
> Eclipse,

Do you mean Eclipse Che?

> but we will have to look at a way for multiple users to access it

Why?

> and keep their work private.

**Comment by WaltRCodes** [comment]
Yes I meant Che. I mean a way for the teacher and student to connect.

**Comment by khatchad** [comment]
Not for this phase.

**Comment by khatchad** [comment]
Beginner programming classes normally don't have a whole bunch of group work. I would suggest a collaborative environment for phase 2.
